build-tools/packages/build-cli/test/commands/generate/buildVersion.test.ts

```diff
@@ -409,26 +409,27 @@ describe("generate:buildVersion", () => {
 			expect(ctx.stdout).to.contain("isLatest=false");
 		});
 
-	test
-		.env({
-			VERSION_BUILDNUMBER: "212045",
-			VERSION_TAGNAME: "client",
-			TEST_BUILD: "false",
-			VERSION_RELEASE: "release",
-			VERSION_INCLUDE_INTERNAL_VERSIONS: "False",
-		})
-		.stdout()
-		.command([
-			"generate:buildVersion",
-			"--fileVersion",
-			"2.0.0-rc.3.0.0",
-			"--tags",
-			...test_tags,
-		])
-		.it("RC version, release", (ctx) => {
-			expect(ctx.stdout).to.contain("version=2.0.0-rc.3.0.0");
-			expect(ctx.stdout).to.contain("isLatest=false");
-		});
+	// Commenting this out because I am not able to patch RC3 https://github.com/microsoft/FluidFramework/pull/21112
```

Comment:
```suggestion
	// Commenting this out because I am not able to patch RC3 https://github.com/microsoft/FluidFramework/pull/21112
	// Seems like it fails because 2.0.0-rc.3.0.0 already exists in npm.
	// We don't release build-tools from release branches, only the client release group, so this should be fine.
```